### PR TITLE
Documentation for rect

### PR DESCRIPTION
**Pull request by dogunbound**
There are more usage examples and tests for the rect.rs module now.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,8 +4,10 @@
 
 ### Added
 - Examples for Vector3
+- Examples for Vector2
```
[the remark this answers is not in the document]
Comment (dogunbound) [reply CHANGELOG.md:7]: That wasnt added, so I thought it be best to add it.

File: src/graphics/rect.rs [hunk 2]
```diff
@@ -41,6 +48,17 @@ impl<T> Rect<T> {
     }
 
     /// Construct a rectangle from its position and size.
+    ///
+    /// # Usage Example
+    ///
+    /// ```
+    /// # use sfml::graphics::Rect;
+    /// # use sfml::system::Vector2i;
+    /// let a = Vector2i::new(10, 20);
```
[the remark this answers is not in the document]
Comment (dogunbound) [reply src/graphics/rect.rs:57]: If you think vector2 will be better, I'll change it to that.

File: CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (crumblingstatue) [reply CHANGELOG.md:7]: Alright

File: src/graphics/rect.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (crumblingstatue) [reply src/graphics/rect.rs:57]: Alright, it doesn't matter too much, but I think it would look better.

[the remark this answers is not in the document]
Comment (dogunbound) [reply src/graphics/rect.rs]: Alright, a new change has pushed that changes it over to Vector2.

**Comment by crumblingstatue** [comment]
Thanks for your contribution!

**Pull request merged**